packages/structures/src/components/component-detail/edit-container.js

```diff
@@ -39,7 +42,122 @@ const ViewContainer = props => {
 
 	useEffect(() => {
 		const getComponent = id
-			? api.getMutualizedComponent(id)
+			? //? api.getMutualizedComponent(id)
+			  Promise.resolve([
+					{
+						labelLg1: 'Actif',
```

Comment:
maybe data shouldn't be in the component

Reply:
oups :S